### PR TITLE
Add Mango Mail as a service provider

### PR DESCRIPTION
**Pull request by MrExcitingDigits**

### CODE REVIEW

File: mymangomail.com.email.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      "host":"@",
+      "data":"mail-host-verify=%verification%",
+      "ttl":3600
+    },
+
+    {
+      "groupId":"setup",
+      "type":"SPFM",
+      "host":"@",
+      "spfRules":"v=spf1 include:spf.mymangomail.com -all",
```
Comment (pawel-kow) [inline mymangomail.com.email.json:27]: For SPFM "v=spf1" and "-all" must not be set

File: mymangomail.com.email.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      "ttl":3600
+    },
+
+    {
+      "groupId":"setup",
+      "type":"TXT",
+      "host":"_dmarc",
+      "data":"v=DMARC1; p=quarantine;",
+      "ttl":3600
```
Comment (pawel-kow) [inline mymangomail.com.email.json:36]: For DMARD it's recommended to set `txtConflictMatchingMode`=`All`

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: mymangomail.com.email.json (hunk 2 above)
Comment (MrExcitingDigits) [reply mymangomail.com.email.json]: Fixed this too.

**Comment by MrExcitingDigits** [comment]
Thanks for the edits, just made the revisions right now.

**Review by pawel-kow — APPROVED** [review]
lgtm

**Pull request merged**